### PR TITLE
Three additions and two fixes for missing EU values

### PR DESCRIPTION
**Pull request by theCapypara**
This adds WONDER_GUMMI_IQ_GAIN and SCRIPT_VARS_VALUES and fixes some missing length values for EU.

### CODE REVIEW

File: symbols/arm9.yml [hunk 1]
```diff
@@ -724,6 +724,14 @@ arm9:
       length:
         NA: 0x4
         EU: 0x4
+    - name: WONDER_GUMMI_IQ_GAIN
+      address:
+        NA: 0x20A1890
+        EU: 0x20A1E14
+      length:
+        NA: 0x1
```
Comment (UsernameFodder) [inline symbols/arm9.yml:732]: It looks like this value's accessed with `ldrsh`, so let's change the length to `0x2`.

File: symbols/ram.yml [hunk 2]
```diff
@@ -223,6 +223,15 @@ arm9:
...
         This is presumably part of a larger struct, together with other nearby data.
+    - name: SCRIPT_VARS_VALUES
+      address:
+        NA: 0x22AB0AC
+        EU: 0x22AB9EC
+      length:
+        NA: 0x2B4
+        EU: 0x2B4
+      description: |-
+        The table of game variable values. It's structure is determined by SCRIPT_VARS.
```
Comment (UsernameFodder) [inline symbols/ram.yml:234]: "Its" with no apostrophe

**Review by UsernameFodder — CHANGES_REQUESTED** [review]

**Comment by UsernameFodder** [comment]
@theCapypara this LGTM, but I suggested some `pmdsky-debug` changes on https://github.com/SkyTemple/skytemple-files/pull/204. Do you want to just pile them onto this PR?

**Comment by theCapypara** [comment]
Sure, I'll add them!

File: symbols/ram.yml [hunk 3]
```diff
@@ -223,6 +223,15 @@ arm9:
         A null-terminated string, with a maximum length of 10. Presumably encoded with the ANSI/Shift JIS encoding the game typically uses.
         
         This is presumably part of a larger struct, together with other nearby data.
+    - name: SCRIPT_VARS_VALUES
```
Comment (UsernameFodder) [inline symbols/ram.yml:226]: [The formatter](https://github.com/UsernameFodder/pmdsky-debug/runs/5266440927?check_suite_focus=true) caught this as being out of order.

**Review by UsernameFodder — CHANGES_REQUESTED** [review]
Could we also add in the description for `TOP_MENU_RETURN_MUSIC_ID` in `literals.yml`? :D

File: symbols/ram.yml (hunk 3 above)
Comment (theCapypara) [reply symbols/ram.yml:226]: The one time I *don't* run it...

Comment (theCapypara) [reply symbols/ram.yml:226]: Oh wait, that's the one I added before, it didn't catch it, I'll check your link and see if I did it wrong.

**Comment by UsernameFodder** [comment]
Bump @theCapypara, this was also from https://github.com/SkyTemple/skytemple-files/pull/204/files:

> Could we also add in the description for TOP_MENU_RETURN_MUSIC_ID in literals.yml?

**Comment by theCapypara** [comment]
I knew I missed something.

**Review by UsernameFodder — APPROVED** [review]
Alright, this all looks good. Thanks for adding these!

**Pull request merged**